src/shared/containers/search-box/SearchBox.js

```diff
@@ -30,6 +31,18 @@ class SearchBox extends Component {
 
     // ---------------------------------------------
 
+    _getRandomSuggestion() {
+        const suggestions = [
```

Comment:
Sure. Since this is only called on mount the difference is small, but makes sense